pandas/core/series.py

```diff
@@ -1446,6 +1446,30 @@ def to_string(self, buf=None, na_rep='NaN', float_format=None, header=True,
     def iteritems(self):
         """
         Lazily iterate over (index, value) tuples.
+
+        This method returns an iterable tuple (index, value). This is
+        convienient if you want to create a a Lazy iterator.
+
+        Returns
+        -------
+        iterable
+            Iterable of tuples containing the (index, value) pairs from a
+            Series.
+
+        See Also
+        --------
+        Series.apply : Invoke function on values of Series.
```

Comment:
Not sure if `Series.apply` and `Series.map` are too related to this method. I would instead reference `Series.items`

Reply:
Unless I am mistaken, it seems to me that Series.iteritems and Series.items are the same method. The docstring of items and iteritems will be the same I don't think that this is a good idea.

I will mention it in the description instead.

Reply:
I agree that apply and map aren't closely enough related to include here - can you remove?